website/docs/releases/linux.md

```diff
@@ -83,6 +83,20 @@ Available via [Personal Package Archive](https://launchpad.net/~feignint/+archiv
 
 Available via [Arch User Repository](https://aur.archlinux.org/packages/dosbox-staging).
 
+### NixOS repository package
+
+Available via NixOS or Home Manager.
+
+    environment.systemPackages = with pkgs; [
```

Comment:
Don't we need some more words around this? (e.g., "Add the following to the xxxx.conf config file:" or something)

Isn't there just a command to do this instead of editing files? (I have no idea, just asking)

Reply:
Yeah, I wrote this presuming some things.

How's this? 

> 
> ### NixOS repository package
> 
> Available via NixOS or Home Manager. Add the following to your `configuration.nix` file:
> 
> For NixOS:
> 
>     environment.systemPackages = with pkgs; [
>       dosbox-staging
>     ];
> 
> For Home Manager:
> 
>     home.packages = with pkgs; [
>       dosbox-staging
>     ];
> 
> Then rebuild your system with: `nixos-rebuild switch`

> Isn't there just a command to do this instead of editing files?

There are a few ways to run stuff with nix. There's the 'traditional' way of installing packages like this: `nix-env -iA nixos.dosbox-staging` which isn't really recommended because it's going against what nix is about. Then there's the ephemeral method like this: `nix run nixpkgs#dosbox-staging` or `nix-shell -p dosbox-staging`. Nix pulls the inputs from the [build script](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/by-name/do/dosbox-staging/package.nix) and puts it in your path. Upon closing the terminal the package is removed from your path. Try it from your machine now--it's pretty neat!

I can add these notes to the install instructions if you wish, but what's really recommended in the nix world is what I already put in the commit.

Reply:
Thanks, I'm fine with this, I just wanted to make the instructions a bit more user-friendly, that's all 😄 Merging then.